data/wof-schema.sql

```diff
@@ -11,3 +12,13 @@ CREATE TABLE wof_neighbourhood (
 CREATE INDEX wof_neighbourhood_wof_id_index ON wof_neighbourhood(wof_id);
 CREATE INDEX wof_neighbourhood_label_position_index ON wof_neighbourhood USING GIST(label_position);
 CREATE INDEX wof_neighbourhood_geometry_index ON wof_neighbourhood USING GIST(geometry);
+
+CREATE TABLE wof_neighbourhood_placetype (
+  placetype_code SMALLINT UNIQUE NOT NULL,
```

Comment:
Is there any reason we'd want to insert multiple records with the same `wof_id` into the database - perhaps to de-multi multipolygons?